wandb/apis/public/api.py

```diff
@@ -421,6 +421,107 @@ def create_run_queue(
             _default_resource_config=config,
         )
 
+    def upsert_run_queue(
+        self,
+        name: str,
+        resource_config: dict,
+        resource_type: "public.RunQueueResourceType",
+        entity: Optional[str] = None,
+        template_variables: Optional[dict] = None,
+        external_links: Optional[dict] = None,
+        prioritization_mode: Optional["public.RunQueuePrioritizationMode"] = None,
+    ):
+        """Upsert a run queue (launch).
+
+        Arguments:
+            name: (str) Name of the queue to create
+            entity: (str) Optional name of the entity to create the queue. If None, will use the configured or default entity.
+            resource_config: (dict) Optional default resource configuration to be used for the queue. Use handlebars (eg. "{{var}}") to specify template variables.
+            resource_type: (str) Type of resource to be used for the queue. One of "local-container", "local-process", "kubernetes", "sagemaker", or "gcp-vertex".
+            template_variables: (dict) A dictionary of template variable schemas to be used with the config. Expected format of:
+                {
+                    "var-name": {
+                        "schema": {
+                            "type": ("string", "number", or "integer"),
+                            "default": (optional value),
+                            "minimum": (optional minimum),
+                            "maximum": (optional maximum),
+                            "enum": [..."(options)"]
+                        }
+                    }
+                }
+            external_links: (dict) Optional dictionary of external links to be used with the queue. Expected format of:
+                {
+                    "name": "url"
+                }
+            prioritization_mode: (str) Optional version of prioritization to use. Either "V0" or None
+
+        Returns:
+            The upserted `RunQueue`.
+
+        Raises:
+            ValueError if any of the parameters are invalid
+            wandb.Error on wandb API errors
+        """
+        if entity is None:
+            entity = self.settings["entity"] or self.default_entity
+            if entity is None:
+                raise ValueError(
+                    "entity must be passed as a parameter, or set in settings"
+                )
+
+        if len(name) == 0:
+            raise ValueError("name must be non-empty")
+        if len(name) > 64:
+            raise ValueError("name must be less than 64 characters")
+
+        if prioritization_mode:
+            prioritization_mode = prioritization_mode.upper()
+            if prioritization_mode not in ["V0"]:
```

Comment:
Why not allow users to create one where prioritization mode is disabled if it doesn't exist?

Reply:
SO if `prioritization is None` we should set the mode to disabled? But default to `V0`? That seems reasonable to me

Reply:
Ah, gotchA! I just copypasta'd this check from `create_run_queue` I assumed @nickpenaranda's intent was to prevent users from creating queues without prioritization so we could just think of those as legacy instead of something we support going forward.